Sources/PointFreeTestSupport/PointFreeTestSupport.swift

```diff
@@ -9,12 +9,12 @@ import Prelude
 extension Environment {
   public static let mock = Environment.init(
     assets: .mock,
-    blogPosts: { [.mock] },
+    blogPosts: unzurry([.mock]),
     cookieTransform: .plaintext,
     database: .mock,
-    date: { .mock },
+    date: unzurry(.mock),
     envVars: .mock,
-    episodes: { [.mock] },
+    episodes: unzurry([.free, .mock]),
```

Comment:
oh just saw this. I think at the very least we wanna swap the `.mock` and `.free` in the array, but also maybe just name them differently and make sure they hold data that is very different from each other.